packages/select/src/components/select/_select.scss

```diff
@@ -3,26 +3,32 @@
 
 @import "../../common/variables";
 
+$select-padding: $pt-grid-size * 0.5;
+
 .#{$ns}-select-popover {
   .#{$ns}-popover-content {
     // use padding on container rather than margin on input group
     // because top margin leaves some empty space with no background color.
-    padding: $pt-grid-size * 0.5;
+    padding: $select-padding;
   }
 
   .#{$ns}-input-group {
     margin-bottom: 0;
   }
 
   .#{$ns}-menu {
+    // offset the padding added by the select componenent so menu can remain full width with padding on either side
```

Comment:
```suggestion
    // offset the padding added by the select componenent so menu can remain full width with padding on either side
    // this is important so menu dividers can be full width, and so that menu items can have padding to the left and right
```